### PR TITLE
chore(inapp): reduce todos, update error message, update tests and add thresholds

### PR DESCRIPTION
**Pull request by Samaritan1011001**
#### Description of changes

This PR has the following changes, 
- Removed TODOs in 2 places
- Updated userPoolId not present error message where previously it was reported as token provider not present
- Added more tests
- Updated the test thresholds

#### Description of how you validated changes
yarn test

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#steps-towards-contributions)
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/auth/__tests__/providers/cognito/identityIdProvider.test.ts [hunk 1]
```diff
@@ -4,16 +4,13 @@
 import { authAPITestParams } from './testUtils/authApiTestParams';
 import { Amplify, Identity, ResourcesConfig } from '@aws-amplify/core';
 import { DefaultIdentityIdStore } from '../../../src/providers/cognito/credentialsProvider/IdentityIdStore';
-
-// TODO(V6): import these from top level core/ and not lib/
-import * as cogId from '@aws-amplify/core/lib/awsClients/cognitoIdentity';
+import * as cogId from '@aws-amplify/core/internals/aws-clients/cognitoIdentity';
 import { cognitoIdentityIdProvider } from '../../../src/providers/cognito/credentialsProvider/IdentityIdProvider';
-jest.mock('@aws-amplify/core/lib/awsClients/cognitoIdentity');
+import { CognitoIdentityPoolConfig } from '@aws-amplify/core/lib-esm/libraryUtils';
```
Comment (cshfang) [inline packages/auth/__tests__/providers/cognito/identityIdProvider.test.ts:9]: I think this should be from the internal utils path?

File: packages/core/internals/aws-clients/cognitoIdentity/package.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+	"name": "@aws-amplify/core/internals/aws-clients/cognitoIdentity",
+	"types": "../../../lib-esm/awsClients/cognitoIdentity/index.d.ts",
+	"main": "../../../lib/awsClients/cognitoIdentity/index.js",
+	"module": "../../../lib-esm/awsClients/cognitoIdentity/index.js",
+	"react-native": "../../../lib-esm/awsClients/cognitoIdentity/index.js",
+	"sideEffects": false
+}
```
Comment (cshfang) [inline packages/core/internals/aws-clients/cognitoIdentity/package.json:8]: Nit: Missing EOF newline

File: packages/core/src/singleton/Auth/utils/errorHelpers.ts [hunk 3]
```diff
@@ -21,6 +22,11 @@ const authConfigurationErrorMap: AmplifyErrorMap<AuthConfigurationErrorCode> = {
 		recoverySuggestion:
 			'Make sure to call Amplify.configure in your app with UserPoolId and IdentityPoolId.',
 	},
+	[AuthConfigurationErrorCode.AuthUserPoolException]: {
+		message: 'Auth UserPool not configured.',
+		recoverySuggestion:
+			'Make sure to call Amplify.configure in your app with UserPoolId.',
```
Comment (elorzafe) [inline packages/core/src/singleton/Auth/utils/errorHelpers.ts:28]:
```suggestion
			'Make sure to call Amplify.configure in your app with userPoolId and userPoolClientId',
```

**Review by elorzafe — APPROVED** [review]
Looks good

Thanks @Samaritan1011001 🎖️

**Review by cshfang — APPROVED** [review]

**Review by elorzafe — APPROVED** [review]

**Review by cshfang — APPROVED** [review]

**Pull request merged**